### ISSUE TITLE
Error Failed to authenticate

### ISSUE THREAD

**Issue by pixelwiz** [rohmanhm/new-egghead-downloader#2]
Hi, I have a working Pro account.  But when I try to put in my email and password I get error failure to authenticate.  Does it require any kind of quotes?  What's the correct syntax for entering in the options?

```bash
ehd -e myemail@email.com -p mypassword -c https://egghead.io/courses/introduction-to-node-the-fundamentals D:\Videos\egghead\NodeFundamentals
```

**Comment by rohmanhm** [comment]
Hi, I can't reproduce your issue.
Maybe you forgot your password.

You don't need require any quotes. 
Example:
```bash
ehd -e email@myemail.com -p mypassword https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx manage-complex-state-in-react-apps-with-mobx
```

Feel free to open this issue if something went wrong

**Comment by pixelwiz** [comment]
That's strange.  The password is correct, cause I was able to login to my pro account.  Further, I actually changed it just to be sure the special characters in it weren't causing a problem with your script.  So I tried it was the new password and got the same error.  I was trying to download an entire course https://egghead.io/courses/introduction-to-node-the-fundamentals and I thought the last parameter had to be a path to a local folder, but in your example it's just a string?
